save/record_instancereference.ts

```diff
@@ -1,7 +1,22 @@
-import getProps from "./properties";
+import { textSpanIsEmpty } from "typescript";
+import getProps, { Property, PropertyCollection } from "./properties";
 import Record from "./record";
 import { SaveBuffer } from "./util";
 
+export class InventoryItem{
+    iref: number;
+    stackedItemsNum: number;
+    changedEntriesNum: number;
+    changedEntries: PropertyCollection[];
+
+    constructor(iref: number, stackedItemsNum: number, changedEntriesNum: number, changedEntries: PropertyCollection[]){
+        this.iref = iref;
+        this.stackedItemsNum = stackedItemsNum;
+        this.changedEntriesNum = changedEntriesNum;
+        this.changedEntries = changedEntries;
+    }
+}
```

Comment:
Since this is being referenced in two `Record` classes, it should probably be in its own .ts file.